pandas/tools/util.py

```diff
@@ -61,6 +61,26 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : {None, 'integer', 'signed', 'unsigned', 'float'} , default None
+        If not None, and if the data has been successfully cast to a
```

Comment:
not what I mean

I might use to_numeric with a numeric series to start, 
one that I want to downcast

it doesn't have to start as object at all that's the point